src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -62,11 +62,13 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit id: String): Try[Unit] = {
     log.info(s"[$id] Finalizing deposit")
+    baseDir = new File(SwordProps("basedir"))
```

Comment:
Oh, wait, you only use it in `getBagDir(tempDir)` and there it turns out to be the `depositDir`. Is this a naming issue or what? `tempDir` does not really sound like the directory where the actual deposit is located...